src/scripts/main.js

```diff
@@ -21,7 +21,19 @@ const food = {
 const tree = document.querySelector('#tree');
 
 function createTree(element, data) {
-  // WRITE YOUR CODE HERE
+  const newTree = document.createElement('ul');
+
+  for (const key in data) {
+    const listItem = document.createElement('li');
+
+    listItem.textContent = key;
+
+    if (Object.keys(data[key]).length > 0) {
+      createTree(listItem, data[key]);
```

Comment:
There is a logical error here. The `createTree` function should append the newly created subtree to the `listItem` instead of just calling `createTree`. You need to append the result of `createTree` to `listItem` to ensure the tree structure is nested correctly.

Reply:
Sorry I don't understand you, what do you mean